Server/service/networkService.js

```diff
@@ -103,20 +103,25 @@ class NetworkService {
       const response = await ping.promise.probe(job.data.url);
       return response;
     };
-
+  
     let isAlive;
-
+  
     try {
-      const { responseTime, response } =
-        await this.measureResponseTime(operation);
+      const { responseTime, response } = await this.measureResponseTime(operation);
       isAlive = response.alive;
+  
+      // Fetch TTL setting from the monitor
+      const monitor = await this.db.getMonitorById(job.data._id);
+      const ttlDays = monitor.ttlDays || 30; // Default to 30 days if not set
```

Comment:
How about we name this `checkTTL` since it's TTL for the check, not the monitor, and we don't want to lock ourselves in to using days, so we should drop the days suffix

Reply:
Agreed. Thank you!